LiskArchive/lisk-elements
# lisk-transaction:validation:schema senderId, recipientId are not required and senderSecondPublicKey is an invalid property - Closes #1121

### What was the problem?

Schema validation was expecting recipentId, SenderId and validating secondPublicKey which were invalid

### How did I fix it?

Removing said fields from schema

### How to test it?

- Build should be green
- Try to register a multi-signature transaction and should be possible

### Review checklist

* The PR resolves #1121
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
